caddytls/client.go

```diff
@@ -331,6 +331,9 @@ func (c *ACMEClient) Renew(name string) error {
 		return errors.New("too many renewal attempts; last error: " + err.Error())
 	}
 
+	// Executes Cert renew events
+	caddy.EmitEvent(caddy.CertRenewEvent, nil)
```

Comment:
What should we pass as data in the second argument?

Reply:
Nothing

Reply:
But won't hooks want to know which certificate was renewed, at least?

Reply:
Great idea!